dom/observable/tentative/observable-ctor.any.js

```diff
@@ -0,0 +1,685 @@
+test(() => {
```

Comment:
Can you also add a test where we assert that calling `new Observable` with _no_ subscriber callback throws an exception, given the WebIDL semantics?

Reply:
I've done this.